app/lib/ga-log-adapter.js

```diff
@@ -33,9 +64,10 @@ class GALogAdapter {
   onForget(keyList) {
     keyList.forEach((key) => {
       if (Object.keys(dimensionMap).includes(key)) {
-        this.layer()('set', `dimension${dimensionMap[key]}`, null);
         if (key === 'projectToken') {
           this.layer()('set', `dimension${dimensionMap[key]}`, 'zooHome');
```

Comment:
should `'zooHome'` be `key` if it's `zooTalk` or something else?
somewhat related to comment regarding `enabledTokens` in `zooniverse-logging.js`.

Reply:
No; this simply says that if we tell it to forget what project we're on and don't give it a new project name, it should assume zooHome as a default.